platform/security-filter-api/src/main/java/org/codice/ddf/platform/filter/HttpFilter.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.platform.filter;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * An HttpFilter is an object that performs filtering tasks on either the request to a resource (a
+ * servlet or static content), or on the response from a resource, or both.
+ *
+ * <p>It is modelled after {@link javax.servlet.Filter} but its {@code doFilter} method accepts
+ * {@link HttpServletRequest} and {@link HttpServletResponse} arguments instead.
+ */
+public interface HttpFilter {
```

Comment:
❓ Why is this part of `security-filter-api` instead of part of `platform-paxweb-jettyconfig` or some other Jetty or HTTP concerned bundle?  This interface does not seem related to security as it is.

Reply:
Yeah I guess it's not necessarily related to security. I put it here since most of the filter implementations are security-related, but it may make more sense in a new api bundle.

I don't think it should go into `platform-paxweb-jettyconfig` since that would pull in a bunch of other transitive dependencies as well.

Reply:
A new API bundle would be safest.  Currently `platform-api` looks pretty safe but that runs the risk of being exposed to other unrelated dependencies in the future.  Already it depends on `javax.validation`.

Reply:
Ok I'll create a new bundle